atomic_cache.gemspec

```diff
@@ -13,23 +13,28 @@ Gem::Specification.new do |spec|
   spec.summary       = 'summary'
   spec.description   = 'desc'
 
-  spec.licenses      = ['apache 2.0']
+  spec.licenses      = ['Apache-2.0']
   spec.homepage      = 'https://github.com/ibotta/atomic_cache'
 
-  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
-    f.match(%r{^(test|spec|features)/})
-  end
+  spec.files             = %w( README.md LICENSE )
+  spec.files            += Dir.glob("lib/**/*")
+  spec.files            += Dir.glob("docs/**/*")
+  spec.files            += Dir.glob("spec/**/*")
 
   spec.require_paths = ['lib']
 
   # Dev dependencies
   spec.add_development_dependency 'bundler', '~> 1.14'
+  spec.add_development_dependency 'gems', '~> 1.0'
+  spec.add_development_dependency 'git', '~> 1.3'
+  spec.add_development_dependency 'github_changelog_generator', '>= 1.15.0.pre.rc'
+  spec.add_development_dependency 'octokit', '~> 4.0'
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'simplecov', '~> 0.15'
   spec.add_development_dependency 'timecop', '~> 0.8.1'
 
   # Dependencies
-  spec.add_dependency 'activesupport', '>= 4.2'
+  spec.add_dependency 'activesupport', '>= 4.2', '< 6'
```

Comment:
binding slightly.

Reply:
it doesn't look like we currently have matrix testing against different rails versions here.

Reply:
Good point - I'll file a story for that. It feels slightly out of scope for the publishing piece. Definitely before we remove the `rc` indicator we'll want this. Thanks for calling this out.